src/web/modules/statistics/performance.http.ts

```diff
@@ -1,55 +1,63 @@
-import { Request, Response } from 'express'
+import {Request, Response} from 'express'
 import moment from 'moment'
-import { Service } from '../../../lib/pay-request/types/adminUsers'
-import { getLiveNotArchivedServices } from '../services/getFilteredServices'
-import { Ledger } from '../../../lib/pay-request'
+import {Service} from '../../../lib/pay-request/types/adminUsers'
+import {getLiveNotArchivedServices} from '../services/getFilteredServices'
+import {Ledger} from '../../../lib/pay-request'
+
+function convertToUnits(value: number) {
+    let stringOfAmount
+
+    if (value >= 1000000 && value < 1000000000) {
+        stringOfAmount = (value / 1.0e6).toFixed(1) + " million";
+    } else if (value >= 10000000) {
+        stringOfAmount = (value / 1.0e9).toFixed(1) + " billion";
+    } else
+        stringOfAmount = value.toString()
 
-function displayAsMillion(amount: number) {
-    const stringOfAmount = (amount/1.0e6).toFixed(1) + " million";
     return stringOfAmount;
 }
 
 export async function overview(req: Request, res: Response) {
-  res.render('statistics/performancePage')
+    res.render('statistics/performancePage')
 }
 
 export async function downloadData(req: Request, res: Response) {
-  const services = await getLiveNotArchivedServices()
-  const countBySector = services.reduce((aggregate: { [key: string]: number; }, service: Service) => {
-    if (service.sector) {
-      const sector = service.sector.replace(' ', '')
-      aggregate[sector] = aggregate[sector] + 1 || 1
-    }
-    return aggregate
-  }, {})
-  const countByOrganisation = services
-    .reduce((aggregate: { [key: string]: number; }, service: Service) => {
-      const org = service.merchant_details && service.merchant_details.name
-      if (org) {
-        aggregate[org] = aggregate[org] + 1 || 1
-      }
-      return aggregate
-    }, {})
-  const orderedCountByOrganisation = Object.keys(countByOrganisation)
-    .sort()
-    .reduce((aggregate: { [key: string]: number; }, organisation: string) => {
-      aggregate[organisation] = countByOrganisation[organisation]
-      return aggregate
+    const services = await getLiveNotArchivedServices()
```

Comment:
Very small thing but the rest of the code is formatted with two spaces rather than 4, this whitespace change is inconsistent and confuses the diff a bit. 

I'm a bit confused as to why the linter is so quiet 🤔 might need to look into that as it shouldn't be on the developer to catch those things

Reply:
Didn't realise it is 2 spaces for javascript vs 4 spaces for Java and using IntelliJ doesn't help. Yeah we probably don't have the lint working since we added gds-pre-commit hook. Updated to 2 spaces